### DISCUSSION TITLE
مشکل در اسکن آی پی

### DISCUSSION THREAD

**Discussion by Arashch405** [MortezaBashsiz/CFScanner#619] [General]
سلام وقت به خیر  من از دیروز هر آی
پیی که میزنم  اسکنر ویندوزبا اینکه تمیزه و مشکلی هم نداره اما اسکن نمیاره در حالی که تا دیروز اوکی بوده نمیدونم مشکل از منه  یا خود اسکنر

**Comment by nilofari700** [comment]
پورت ها همراه اول بستن اسکن ظاهرا نمیشه کرد فعلا

**Comment by Arashch405** [comment]
سلام وقت بخیر...همراه اول نخواستم..روایرانسل مشکل داشتم

‫‪nilo‬‏ ***@***.***‬‏> در تاریخ دوشنبه ۱ ژانویه ۲۰۲۴ ساعت
۱۹:۰۰ نوشت:‬
